dockerfiles/entrypoints/postgres.sql

```diff
@@ -0,0 +1,3 @@
+-- Initialization script.
+-- https://github.com/docker-library/docs/blob/master/postgres/README.md#initialization-scripts
```

Comment:
this is only executed the first time the container is created, if you already have one and don't want to re-create it, just create the db manually from the shell.

Reply:
This comment should be in the code :)